easybuild/easyblocks/n/numpy.py

```diff
@@ -244,8 +245,9 @@ def sanity_check_step(self, *args, **kwargs):
         }
         custom_commands = [
             ('python', '-c "import numpy"'),
-            ('python', '-c "import numpy.core._dotblas"'),  # _dotblas is required for decent performance of numpy.dot()
         ]
+        if LooseVersion(self.version) < LooseVersion("1.10"):
+            custom_commands.append ('python', '-c "import numpy.core._dotblas"')  # _dotblas is required for decent performance of numpy.dot()
```

Comment:
@gppezzi: there's no equivalent check that can be done for recent numpy versions? any reference to documentation on this change?